lab_6_pipeline/pipeline.py

```diff
@@ -96,11 +163,22 @@ def get_cleaned_sentence(self) -> str:
         """
         Returns the lowercase representation of the sentence
         """
+        sentence = []
+        for token in self._tokens:
+            sentence.append(token.get_cleaned())
```

Comment:
if cleaned version is empty string, do you want to add it as well?

Reply:
No, i don't need it
I added the check